server/api/__init__.py

```diff
@@ -12,3 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+# Minimum client version that supports model monitoring,
+# Will be fixed when MM will be defined as BC supported feature
+MINIMUM_CLIENT_VERSION_FOR_MM = "1.7.0"
```

Comment:
don't you prefer to define it in a more model-monitoring specific place like server/api/api/endpoints/model_monitoring.py?

Reply:
We need this in few more places so when  I tried to put it in where I got some import issues.